fact/analysis/statistics.py

```diff
@@ -39,3 +79,100 @@ def li_ma_significance(n_on, n_off, alpha=0.2):
         return significance[0]
 
     return significance
+
+
+def calc_weight_simple_to_curved(energy, gamma, a, b):
+    '''
+    Reweigt simulated events from a simulated power law with
+    spectral index `gamma` to a curved power law with parameters `a` and `b`
+
+    Parameters
+    ----------
+    energy: float or array-like
+        Energy of the events
+    gamma: float
+        Spectral index of the simulated power law
+    a: float
+        Parameter `a` of the target curved power law
+    b: float
+        Parameter `b` of the target curved power law
+    '''
+    return energy ** (gamma + a + b * np.log10(energy))
+
+
+def calc_weight_change_index(energy, simulated_gamma, target_gamma):
+    '''
+    Reweigt simulated events from one power law index to another
+
+    Parameters
+    ----------
+    energy: float or array-like
+        Energy of the events
+    simulated_gamma: float
+        Spectral index of the simulated power law
+    target_gamma: float
+        Spectral index of the target power law
+    '''
+    return energy ** (target_gamma - simulated_gamma)
+
+
+def calc_gamma_obstime(n_events, gamma, phi_0, max_impact, e_min, e_max):
+    '''
+    Calculate the equivalent observation time for a gamma montecarlo set
```

Comment:
`gamma` refers sometimes to the particle type, sometimes to the spectral index ... proposal use `spectral_index` over `gamma`, and `flux_normalisation` over `phi_0` maybe.